examples/01b-Getting-started-Pytorch.ipynb

```diff
@@ -335,7 +335,9 @@
     "optim = Adam\n",
```

Comment:
<p>Why do we have TensorFlow in this sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Looks like a typo. Updated to PyTorch</p>